content/blog/breaks/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Breaks
+date: "2019-04-16T22:12:03.284Z"
+description: Replenish your energy.
+---
+
+> A break at work is a period of time during a shift in which an employee is allowed to take time off from his/her job. (Wikipedia).
+
+### Guilt
+
+Workers often feel guilty if they stand up and take a break. It's especially true for people in tech and the newcomers like interns or junior programmers. Hustling hard is the new norm, and I don't have anything against hard work, but play it smart to avoid getting [burned out](https://devhealth.io/burnout/).
```

Comment:
_Hustling hard is the new norm, and I don't have anything against hard work, but play it smart to avoid getting burned out._

I would put it like that:

Hustling hard is the new norm. Not that I have anything against hard work, but you should play it smart to avoid getting burned out. 

or, if you want one sentence:

Hustling hard is the new norm, and not that I have anything against hard work, but you should play it smart to avoid getting burned out.